doc_src/cmds/_.rst

```diff
@@ -0,0 +1,34 @@
+.. _cmd-_:
+
+_ - call fish's translations
+============================
+
+Synopsis
+--------
+
+::
+
+    _ STRING...
+
+Description
+-----------
+
+``_`` is used to let fish translate a string to another language. This can only be used to look up fish's own translations, and it requires fish to be built with gettext support, or it will simply echo the strings back.
```

Comment:
How about

```
``_`` translates its arguments into the current language, if possible.

It is equivalent to ``gettext fish STRING``, meaning it can only be used to look up fish's own translations.

It requires fish to be built with gettext support. If that support is disabled, or there is no translation it will simply echo the argument back.

The language depends on the current locale, set with ``$LANG`` and ``$LC_MESSAGES``.
```

Reply:
I've been trying to get away from that "professional technical documentation" style of writing a bit.

It can be super dense and, especially for non-native speakers, hard to read.